infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/infrastructure/plugins/DefaultClusterManager.java

```diff
@@ -41,28 +57,72 @@
 
     private static final String ACTIVE_CLUSTER_NODE = "primary_site_node";
     private static final String STANDBY_CLUSTER_NODE = "standby_site_node";
-    private boolean ifShutdown = false;
+
+
+    public static final String CONFIG_NAMESPACE = "ns_lr_config_it";
+    public static final String CONFIG_TABLE_NAME = "lr_config_it";
+    public static final CommonTypes.Uuid OP_RESUME = CommonTypes.Uuid.newBuilder().setLsb(0L).setMsb(0L).build();
+    public static final CommonTypes.Uuid OP_SWITCH = CommonTypes.Uuid.newBuilder().setLsb(1L).setMsb(1L).build();
```

Comment:
What are these OP? can we have a general description?

Reply:
It is operations. I think it is kind of ugly to hardcode it and I will try to directly write new config to tables.